backend/graph/schema.graphqls

```diff
@@ -12,13 +12,17 @@ directive @goField(
 type Query {
   processes: [Process!]!
   process(processKey: Int!): Process
+  instances: [Instance!]!
+  instance(instanceKey: Int!): Instance
 }
 
 type Process {
+  activeInstances: Int!
+  completedInstances: Int!
   bpmnLiveStatus: String!
   bpmnResource: String!
   deploymentTime: DateTime!
-  instances: [Instance!]! @goField(forceResolver: true)
```

Comment:
@kauppie if I remembered correctly, I have to remove @goField here so that the instance list can be queried. It's a bit weird that I put it back now and retry, querying seems to work.